test/countConsonants.test.js

```diff
@@ -15,9 +16,8 @@ describe("countConsonants", () => {
     expect(countConsonants("")).toEqual(0);
   });
   test("invalid parameters", () => {
-    expect(() => countConsonants({obj:"val"})).toThrowError(TypeError);
-    expect(() => countConsonants(["array"])).toThrowError(TypeError);
-    expect(() => countConsonants(undefined)).toThrowError(TypeError);
-    expect(() => countConsonants(null)).toThrowError(TypeError);
+    testTypes
+      .filter(t => typeof t != "string")
+      .forEach(t => expect(() => countConsonants(t)).toThrowError(TypeError));
```

Comment:
I think I would rather the tests be explicit. The types array could be updated separately and change how this test works.